angular/src/app/account/send-sidechain/sending/send-sending.component.ts

```diff
@@ -44,6 +43,15 @@ export class AccountSendSidechainSendingComponent implements OnInit, OnDestroy {
 
     } else {
       this.sendService.transactionId = this.sendService.transactionResult;
+
+      // This means the transaction was sent successfully, we should now mark the UTXOs as spent.
+      this.sendService.selectedData.inputs.forEach((i) => {
```

Comment:
Will this get refresh to reflect unconfirmed?
What if a trx is dropped form mempool will they become spent again or de we need a resync?

Reply:
This "spent" status is only valid until next full sync, which happens within a minute, or if there are actual changes to the UTXOs in data received during watch. Not 100% sure if user can come in a scenario where they need to wait (or manually run resync button).